### ISSUE TITLE
Brave opens various connections with no website opened 

### ISSUE THREAD

**Issue by Wend1go** [brave/brave-browser#6963]
Some guys on the "Heise" forum have discovered that a freshly installed Brave browser (no settings changed) tries to contact these IPs/Domains even with no website opened:
```
172.217.21.195    Google
40.115.22.134     MSFT - Microsoft
104.28.23.242     Cloudflare
52.87.74.158      Amazon
151.101.1.7       SKYCA-3 - Fastly
151.101.113.7     SKYCA-3 - Fastly
151.101.13.7      SKYCA-3 - Fastly
151.101.2.49      SKYCA-3 - Fastly
151.101.37.7      SKYCA-3 - Fastly
static.88-99-170-33.clients.your-server.de/
mil04s23-in-f110-1e100.net
fra16s18-in-f4.1e100.net
```
https://www.heise.de/forum/heise-online/News-Kommentare/Brave-1-0-Browser-blockiert-Werbung-und-Tracker/Hintergrundverbindungen-zu-Google-Microsoft-Amazon-Fastly-Cloudflare/posting-35609319/show/
(german)

What is the purpose / are the connections necessary?

**Comment by bsclifton** [comment]
cc: @jonathansampson (who has looked into this before; see https://brave.com/brave-tops-browser-first-run-network-traffic-results/)

**Comment by Wend1go** [comment]
Thanks for the link.
Looks like the above mentioned URLs were not mentioned in the brave traffic results.

**Comment by jonathansampson** [comment]
@Wend1go I suspect something is making those calls. Checking the traffic in Fiddler ([as I did](https://brave.com/brave-tops-browser-first-run-network-traffic-results/)) would show you which processes is making those specific calls. When I wrote my post on Network Activity, I included all calls that Brave makes.

If the user is correct, and their Brave is making additional calls, I would ask where they downloaded Brave (is it possible they got an infected version?), who manages their computer (is it possible an Admin auto-installs extensions?), etc.
